.github/dependabot.yml

```
@@ -0,0 +1,29 @@
+    allow:
+      - dependency-type: "all"
+    # Group updates to reduce noise
+    groups:
+      security-updates:
+        applies-to: security-updates
+        patterns:
+          - "*"
+      dev-dependencies:
+        dependency-type: "development"
+        patterns:
+          - "*"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)